### DISCUSSION TITLE
Operate YouTube Channel with fabric and yutu

### DISCUSSION THREAD

**Discussion by OpenWaygate** [danielmiessler/Fabric#982] [Show and tell]
Hi all, I'd like to show [yutu🐰](https://github.com/eat-pray-ai/yutu), which can cooperate with fabric!

[yutu🐰](https://github.com/eat-pray-ai/yutu) is a fully functional CLI for YouTube, which allows you upload videos to YouTube Channel, manage playlist, comment on video and so on, without leaving your favorite terminal, or even in pipeline!

More than grab contents from YouTube, [yutu🐰](https://github.com/eat-pray-ai/yutu) also enable you, or even fabric, to operate your YouTube Channel automatically!

Here is a fully automated, serverless YouTube Channel demonstrated as a POC. All contents and operations are fulfilled by yutu, like video uploading, leaving comments, rating and add videos to playlist, anything you want!

https://www.youtube.com/@words-everyday

Moreover, these two GitHub Actions boost your workflow vastly!
1. [eat-pray-ai/youtube-action](https://github.com/eat-pray-ai/youtube-action)
2. [eat-pray-ai/youtube-uploader](https://github.com/eat-pray-ai/youtube-uploader)

**Comment by isudoajl** [comment]
Hello, after the go fabric update, I just figure out that yt command it's not present. There is something wrong? It's not anymore built-in into fabric?

**Reply by OpenWaygate** [reply]
As per the documentation, you should use `-y` now.

https://github.com/danielmiessler/fabric/blob/14d02073abf43ecaeca917f9b314df402b1d73b6/README.md?plain=1#L237-L239
